examples/_d2n-std/nix/app/packages.nix

```diff
@@ -0,0 +1,22 @@
+{
+  inputs,
+  cell,
+}: {
+  default = cell.packages.app;
+  app =
+    # a terrible idea
+    (inputs.dream2nix.lib.makeFlakeOutputs {
+      systems = [inputs.nixpkgs.system];
+      source = inputs.src;
+      projects = {
+        prettier = {
+          name = "prettier";
+          subsystem = "nodejs";
+          translator = "yarn-lock";
+        };
+      };
+    })
+    .packages
+    .${inputs.nixpkgs.system}
+    .default;
```

Comment:
You might want to pick `prettier` here instead of default. Sometimes the `default` might not point to what you want. The heuristic is a bit poor as of now.